src/features/step_definitions/upload-documents.steps.js

```diff
@@ -5,6 +5,8 @@ const managementPlanUploadPage = require("../page_objects/management_plan/manage
 const managementPlanCheckPage = require("../page_objects/management_plan/management-plan-check.page");
 const legalAgreementUploadPage = require("../page_objects/legal_agreement/legal-agreement-upload.page");
 const legalAgreementCheckPage = require("../page_objects/legal_agreement/legal-agreement-check.page");
+const landBoundaryFileUploadPage = require("../page_objects/land_boundary/image-file.page");
```

Comment:
Do you have a standard for what the file is named compared to the class/constant? some match some don't

Reply:
design haven't finalised the naming convention yet... will tidy up when they do